<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleTestsWidget</class>
 <widget class="QWidget" name="ScheduleTestsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #e6f0fa; /* Fond bleu tr√®s clair */
    font-family: 'Segoe UI', sans-serif;
    font-size: 14px;
}
</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Main title */
QLabel#titleLabel {
    font-size: 24px;
    font-weight: bold;
    color: #1F3C88;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, 
                              stop:0 rgba(255,255,255,0.9), 
                              stop:1 rgba(245,249,255,0.8));
    border-bottom: 2px solid #2A5D9F;
    padding: 12px 20px;
    margin-bottom: 15px;
}</string>
   </property>
   <property name="text">
    <string>Schedule a New Test</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>601</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>541</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QDateEdit" name="testDateEdit">
       <property name="styleSheet">
        <string notr="true">QDateEdit {
    background-color: white;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 4px;
    font-size: 13px;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="testNameLineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
    background-color: white;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 4px;
    font-size: 13px;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="styleSheet">
        <string notr="true">QLabel#dateLabel {
    font-size: 18px;
    font-weight: bold;
    color: #113f7c; /* Bleu profond */
}
</string>
       </property>
       <property name="text">
        <string>    Test Date :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#saveButton {
	background-color: #2a5d9f;
    padding: 8px 16px;
    border-radius: 6px;
	font-size: 18px;
    font-weight: bold;
    color: white 
}

QPushButton#saveButton:hover {
    background-color: #1f497d;
}

</string>
       </property>
       <property name="text">
        <string>Save Test</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="backButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#backButton {
	background-color: #2a5d9f;
    padding: 8px 16px;
    border-radius: 6px;
	font-size: 18px;
    font-weight: bold;
    color: white 
}

QPushButton#backButton:hover {
    background-color: #1f497d;
}

</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="styleSheet">
        <string notr="true">QLabel#nameLabel {
    font-size: 18px;
    font-weight: bold;
    color: #113f7c; /* Bleu profond */
}
</string>
       </property>
       <property name="text">
        <string>    Test Name :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label {
    font-size: 18px;
    font-weight: bold;
    color: #607D8B
}</string>
    </property>
    <property name="text">
     <string>a New Test</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
